Descent3/init.cpp

```diff
@@ -1413,33 +1424,63 @@ static const tTempFileInfo temp_file_wildcards[] = {{"d3s*.tmp"}, {"d3m*.tmp"},
                                                     {"d3c*.tmp"}, {"d3t*.tmp"}, {"d3i*.tmp"}};
 static const int num_temp_file_wildcards = sizeof(temp_file_wildcards) / sizeof(tTempFileInfo);
 
+
+// Get the binary path
+char exec_path[_MAX_PATH];
+char* getExecutablePath(char *exec_path, size_t len) {
```

Comment:
Better to move it to ddio module.

Reply:
Where would you move it to? I think this gets called before ddio is initialized.

Reply:
ddio_lnx/lnxfile.cpp / ddio_win/winfile.cpp has some generic file / directory operations.